test/e2e/checkpoint_test.go

```diff
@@ -711,12 +711,12 @@ var _ = Describe("Podman checkpoint", func() {
 		session := podmanTest.Podman([]string{"run", "--network=none", "-d", "--rm", ALPINE, "top"})
 		session.WaitWithDefaultTimeout()
 		Expect(session).Should(ExitCleanly())
-		Expect(podmanTest.NumberOfContainersRunning()).To(Equal(1))
+		Expect(podmanTest.NumberOfContainersRunning()).To(Equal(1), "# of running containers at start")
 		cid := session.OutputToString()
 		fileName := filepath.Join(podmanTest.TempDir, "/checkpoint-"+cid+".tar.gz")
 
 		// Change the container's root file-system
-		result := podmanTest.Podman([]string{"exec", cid, "/bin/sh", "-c", "echo test" + cid + "test > /test.output"})
+		result := podmanTest.Podman([]string{"exec", cid, "touch", "/test.output"})
```

Comment:
can you split out `"/test.output"` to a variable and then use it below when it is verified again?
This is super fragile because if anyone typos that name the test is a NOP real quick.